tests/test_misc.py

```diff
@@ -0,0 +1,67 @@
+"""Test the xdem.misc functions."""
+from __future__ import annotations
+
+import warnings
+
+import pytest
+
+import xdem
+import xdem.misc
+
+
+@pytest.mark.parametrize("deprecation_increment", [-1, 0, 1, None])
```

Comment:
so this is the equivalent of 2 independent `for` loops?

Reply:
Yes, so this one evaluates into 12 independent tests!